pkg/machine/hyperv/machine.go

```diff
@@ -683,3 +695,29 @@ func (m *HyperVMachine) setRootful(rootful bool) error {
 	m.HostUser.Modified = true
 	return nil
 }
+
+func (m *HyperVMachine) cleanupGVProxy() {
```

Comment:
Not for this PR, but this is something that should be added to the refactor. I can add this as one of the common functions that are shared so we don't have more duplicates

Reply:
100% agreed